frontend/app/services/user-service.server.ts

```diff
@@ -4,6 +4,9 @@ export const userInfoSchema = z.object({
   firstName: z.string().optional(),
   lastName: z.string().optional(),
   phoneNumber: z.string().optional(),
+  homeAddress: z.string().optional(),
```

Comment:
Do we have any intention on creating an address schema for this sprint to hold fields like address (line), city, postal code, province/territory?

I may or may not have changed the following user story to include the above four fields in the address form but I can change back to a single field per address and leave it for the rest of this sprint:
https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/2767/

Reply:
We'll stick for a single field for both addresses for now. Resolving